src/features/cartography/infrastructure/presentation/directives/leaflet-map-state-change/leaflet-map-state-change.directive.ts

```diff
@@ -0,0 +1,61 @@
+import type { AfterViewInit } from '@angular/core';
+import { Directive, EventEmitter, Output } from '@angular/core';
+import type { BBox } from 'geojson';
+import type { LatLng, LatLngBounds } from 'leaflet';
+
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
+import { LeafletMapComponent } from '../../components';
+
+export interface ViewBox {
+  boundingBox: BBox;
+  zoomLevel: number;
+}
+
+export interface ViewReset extends ViewBox {
+  center: LatLng;
+}
+
+@Directive({
+  selector: 'leaflet-map[stateChange]'
+})
+export class LeafletMapStateChangeDirective implements AfterViewInit {
+  @Output() public readonly viewreset: EventEmitter<ViewReset> = new EventEmitter<ViewReset>();
```

Comment:
```suggestion
  @Output('stateChange') public readonly viewreset: EventEmitter<ViewReset> = new EventEmitter<ViewReset>();
```
Mais le linter va peut-être t'embêter, du coup soit tu le désactive, soit :
```suggestion
  @Output() public readonly stateChange: EventEmitter<ViewReset> = new EventEmitter<ViewReset>();
```